Sources/Core/AWSSDKEventStreamsAuth/Tests/AWSSDKEventStreamsAuthTests/SigV4EventSigningTests.swift

```diff
@@ -0,0 +1,93 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Smithy
+import XCTest
+import SmithyHTTPAuthAPI
+import SmithyHTTPAuth
+import SmithyHTTPAPI
+import SmithyIdentity
+import SmithyTestUtil
+import AwsCommonRuntimeKit
+import SmithyTimestamps
+import SmithyEventStreams
+import SmithyEventStreamsAPI
+import AWSSDKIdentity
+import AWSSDKHTTPAuth
+
+class SigV4EventSigningTests: XCTestCase {
```

Comment:
The tests for SigV4 signing of event stream events has been moved from `AWSSDKHTTPAuth` to `AWSSDKEventStreamsAuth` to break a circular dependency between those two modules.